_inc/lib/tracks/client.php

```diff
@@ -1,130 +0,0 @@
-<?php
-/**
- * PHP Tracks Client
- * @autounit nosara tracks-client
- * Example Usage:
- *
-```php
-	include( plugin_dir_path( __FILE__ ) . 'lib/tracks/client.php');
-	$result = jetpack_tracks_record_event( $user, $event_name, $properties );
-
-	if ( is_wp_error( $result ) ) {
-		// Handle the error in your app
-	}
-```
- */
-
-// Load the client classes
-require_once( dirname(__FILE__) . '/class.tracks-event.php' );
-require_once( dirname(__FILE__) . '/class.tracks-client.php' );
-
-// Now, let's export a sprinkling of syntactic sugar!
-
-/**
- * Procedurally (vs. Object-oriented), track an event object (or flat array)
- * NOTE: Use this only when the simpler jetpack_tracks_record_event() function won't work for you.
- * @param \Jetpack_Tracks_Event $event The event object.
- * @return \Jetpack_Tracks_Event|\WP_Error
- */
-function jetpack_tracks_record_event_raw( $event ) {
```

Comment:
We should definitely deprecate the functions more carefully here, and keep them deprecated for at least 2 versions before removing them.